### ISSUE TITLE
TypeError: undefined is not an object (evaluating 'e.getLength')

### ISSUE THREAD

**Issue by peyanski** [GoogleForCreators/web-stories-wp#3605]
TypeError: undefined is not an object (evaluating 'e.getLength')
    in kH
    in Unknown
    in Dm
    in Unknown
    in OH
    in TH
    in div
    in layout__Area
    in section
    in layout__Editor
    in RH
    in Unknown
    in RN
    in Unknown
    in Ce
    in Unknown
    in RL
    in Unknown
    in sT
    in Unknown
    in oM
    in Unknown
    in bs
    in Unknown
    in xA
    in Unknown
    in Qc
    in Unknown
    in $l
    in QN
    in Se
    in te
    in NH
    in v

**Comment by ernee** [comment]
@peyanski thanks for reporting!

1) Could you tell us the last action you took before the error occurred? 

2) Can you recreate the error and share any screenshots of the issue or [browser console errors](https://i.imgur.com/KyLAzgH.png)?

**Comment by ernee** [comment]
@peyanski I'm following up on your issue to see if you were able to make the checks above? 

Providing your version of WordPress or your [Site Health Information](https://i.imgur.com/Rtkmsba.png) could be helpful too.

**Comment by ernee** [comment]
Hi @peyanski ! just another quick follow up. Have you tried the latest Beta 2 available for download [on the website](https://google.github.io/web-stories-wp/beta/)? If you have a chance, could you try it and let us know if this issue persists? thank you!

**Comment by peyanski** [comment]
Hi, I tried the beta 2, but I return back the beta 1 as some of the slides were missing after the update. I faced the reported issue only once and never managed to reproduce it again

**Comment by ernee** [comment]
@peyanski thanks so much for this information! 

1) Could you tell us if you were using one of the pre-made templates or did you create a new story?

2) Could you share your [Site Health information](https://i.imgur.com/Rtkmsba.png)? It would be helpful for us to know more about your environment to investigate the issue.

Thank you!

**Comment by peyanski** [comment]
Hi,

1. Both I used templates and create new story
2. Here is the site health info

`
### wp-core ###

version: 5.5.1
site_language: en_US
user_language: en_US
timezone: Europe/Sofia
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 1
default_comment_status: open
environment_type: production
user_count: 2
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /home/u338430817/domains/peyanski.com/public_html
wordpress_size: 52.25 MB (54787750 bytes)
uploads_path: /home/u338430817/domains/peyanski.com/public_html/wp-content/uploads
uploads_size: timeout while calculating size
themes_path: /home/u338430817/domains/peyanski.com/public_html/wp-content/themes
themes_size: 17.15 MB (17986824 bytes)
plugins_path: /home/u338430817/domains/peyanski.com/public_html/wp-content/plugins
plugins_size: 110.63 MB (116003465 bytes)
database_size: 83.79 MB (87855492 bytes)
total_size: not available

### wp-active-theme ###

name: Hestia (hestia)
version: 3.0.4
author: ThemeIsle
author_website: https://themeisle.com
parent_theme: none
theme_features: core-block-patterns, post-thumbnails, title-tag, automatic-feed-links, custom-logo, html5, custom-header, customize-selective-refresh-widgets, custom-background, starter-content, themeisle-demo-import, align-wide, header-footer-elementor, editor-color-palette, menus, editor-style, woocommerce, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, widgets
theme_path: /home/u338430817/domains/peyanski.com/public_html/wp-content/themes/hestia
auto_update: Disabled

### wp-themes-inactive (5) ###

Astra: version: 2.5.4, author: Brainstorm Force,Auto-updates disabled
Twenty Nineteen: version: 1.7, author: the WordPress team,Auto-updates disabled
Twenty Seventeen: version: 2.4, author: the WordPress team,Auto-updates disabled
Twenty Sixteen: version: 2.2, author: the WordPress team,Auto-updates disabled
WPEX Blogger: version: 1.2, author: WPExplorer,Auto-updates disabled

### wp-plugins-active (20) ###

Akismet Anti-Spam: version: 4.1.6, author: Automattic, Auto-updates enabled
Better Click To Tweet: version: 5.9.4, author: Ben Meredith, Auto-updates enabled
Check Email: version: 0.6.2, author: MachoThemes, Auto-updates enabled
Easy Table of Contents: version: 2.0.11, author: Steven A. Zahm, Auto-updates disabled
Google Tag Manager for Wordpress: version: 1.11.4, author: Thomas Geiger, Auto-updates disabled
HurryTimer: version: 2.2.27, author: Nabil Lemsieh, Auto-updates disabled
Loginizer: version: 1.5.7, author: Softaculous, Auto-updates enabled
MakeStories Helper: version: 1.2, author: MakeStories Team, Auto-updates disabled
Newsletter, SMTP, Email marketing and Subscribe forms by Sendinblue: version: 3.0.6, author: Sendinblue (latest version: 3.0.7), Auto-updates disabled
OneSignal Push Notifications: version: 2.1.4, author: OneSignal, Auto-updates disabled
Orbit Fox Companion: version: 2.9.17, author: Themeisle, Auto-updates enabled
Popup Maker: version: 1.11.2, author: Popup Maker, Auto-updates enabled
Rank Math SEO: version: 1.0.47.1, author: Rank Math, Auto-updates disabled
Site Kit by Google: version: 1.15.0, author: Google, Auto-updates enabled
TablePress: version: 1.12, author: Tobias Bäthge, Auto-updates disabled
Web Stories: version: 1.0.0-beta.1, author: Google, Auto-updates disabled
WooCommerce: version: 4.4.1, author: Automattic, Auto-updates disabled
WooCommerce Stripe Gateway: version: 4.5.2, author: WooCommerce, Auto-updates disabled
WPForms Lite: version: 1.6.2.2, author: WPForms, Auto-updates disabled
WP Mail SMTP: version: 2.3.1, author: WPForms, Auto-updates disabled

### wp-media ###

image_editor: WP_Image_Editor_Imagick
imagick_module_version: 1802
imagemagick_version: ImageMagick 7.0.10-10 Q16 x86_64 2020-05-12 https://imagemagick.org
file_uploads: File uploads is turned off
post_max_size: 64M
upload_max_filesize: 64M
max_effective_size: 64 MB
max_file_uploads: 20
imagick_limits: 
	imagick::RESOURCETYPE_AREA: 440 GB
	imagick::RESOURCETYPE_DISK: 9.2233720368548E+18
	imagick::RESOURCETYPE_FILE: 37500
	imagick::RESOURCETYPE_MAP: 440 GB
	imagick::RESOURCETYPE_MEMORY: 220 GB
	imagick::RESOURCETYPE_THREAD: 1
gd_version: 2.2.5
ghostscript_version: 9.25

### wp-server ###

server_architecture: Linux 3.10.0-962.3.2.lve1.5.26.1.el7.x86_64 x86_64
httpd_software: LiteSpeed
php_version: 7.4.8 64bit
php_sapi: litespeed
max_input_variables: 5000
time_limit: 60
memory_limit: 128M
admin_memory_limit: 256M
max_input_time: 60
upload_max_filesize: 64M
php_post_max_size: 64M
curl_version: 7.62.0 OpenSSL/1.0.2k
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: true

### wp-database ###

extension: mysqli
server_version: 10.4.13-MariaDB
client_version: mysqlnd 7.4.8

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /home/u338430817/domains/peyanski.com/public_html/wp-content
WP_PLUGIN_DIR: /home/u338430817/domains/peyanski.com/public_html/wp-content/plugins
WP_MAX_MEMORY_LIMIT: 256M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: false
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_LOCAL_DEV: undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: writable
uploads: writable
plugins: writable
themes: writable

### wpforms ###

version: 1.6.2.2
license: false
lite: Aug 13, 2019 @ 4:02pm
total_forms: 3
total_entries: 101

### google-site-kit ###

version: 1.15.0
php_version: 7.4.8
wp_version: 5.5.1
reference_url: https://peyanski.com
amp_mode: no
site_status: connected-site
user_status: authenticated
active_modules: site-verification, search-console, adsense, analytics, pagespeed-insights
required_scopes: 
	openid: ✅
	https://www.googleapis.com/auth/userinfo.profile: ✅
	https://www.googleapis.com/auth/userinfo.email: ✅
	https://www.googleapis.com/auth/siteverification: ✅
	https://www.googleapis.com/auth/webmasters: ✅
	https://www.googleapis.com/auth/adsense.readonly: ✅
	https://www.googleapis.com/auth/analytics.readonly: ✅
search_console_property: https://peyanski.com/
adsense_account_id: pub-615•••••••••••••
adsense_client_id: ca-pub-615•••••••••••••
adsense_account_status: approved
adsense_use_snippet: yes
analytics_account_id: 3871••
analytics_property_id: UA-3871••••
analytics_profile_id: 2002•••••
analytics_use_snippet: no

### wp_mail_smtp ###

version: 2.3.1
license_key_type: lite
debug: No debug notices found.
db_tables: wpis_wpmailsmtp_tasks_meta

`

On 3 Sep 2020, at 17:37, Renee Johnson <notifications@github.com> wrote:


@peyanski <https://github.com/peyanski> thanks so much for this information!

Could you tell us if you were using one of the pre-made templates or did you create a new story?

Could you share your Site Health information <https://i.imgur.com/Rtkmsba.png>? It would be helpful for us to know more about your environment to investigate the issue.

Thank you!

**Comment by ernee** [comment]
@peyanski we just released version 1.0.0-rc.1 which should be available as an update. Could you try this version and let us know if the missing elements still persist?

**Comment by peyanski** [comment]
Thanks I just update to the rc 1 and everything seems great

On 11 Sep 2020, at 3:00, Renee Johnson <notifications@github.com> wrote:


@peyanski <https://github.com/peyanski> we just released version 1.0.0-rc.1 which should be available as an update. Could you try this version and let us know if the missing elements still persist?

**Comment by ernee** [comment]
@peyanski thanks for the update and glad that fixed it!
